File: Backend/src/main/java/com/feedle/feedleapi/Models/Comment.java
package com.feedle.feedleapi.Models;

import lombok.Data;

/** Represents a comment.
 * Getters/Setters, no argument constructor, equals(), canEqual(),
 * hashCode(), ToString() are available and generated at runtime.
 * @author Marton Pentek
 * @version 1.0
 * @since 12/2020
 */
@Data
public class Comment {
    /**
     * Represents the Id of the comment generally used for accessing the comment.
     */
    public int id;
    /**
     * Represents the actual text in the comment.
     */
    public String content;
    /**
     * Represents the id of the author, used to connect the author to the commment.
     */
    public int userId;
    /**
     * Represents the name of the author, used for display purposes;
     */
    public String authorUserName;
    /**
     * Represents the day of creation.
     */
    public int day;
    /**
     * Represents the month of creation.
     */
    public int month;
    /**
     * Represents the year of creation.
     */
    public int year;
    /**
     * Represents the hour of creation.
     */
    public int hour;
    /**
     * Represents the minute of creation.
     */
    public int minute;
    /**
     * Represents the second of creation.
     */
    public int second;

    /**
     * Represents the post reference
     */
    public int postId;
}

File: Front-end/Feedle/Models/Comment.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Feedle.Models
{
    public class Comment
    {
        [Key]
        [JsonPropertyName("id")]
        public int Id { get; set; }
        [JsonPropertyName("content")]
        public string Content { get; set; }
        [JsonPropertyName("userId")]
        public int UserId { get; set; }
        [JsonPropertyName("authorUserName")]
        public string AuthorUserName { get; set; }
        [JsonPropertyName("day")]
        public int Day { get; set; }
        [JsonPropertyName("month")]
        public int Month { get; set; }
        [JsonPropertyName("year")]
        public int Year { get; set; }
        [JsonPropertyName("hour")]
        public int Hour { get; set; }
        [JsonPropertyName("minute")]
        public int Minute { get; set; }
        [JsonPropertyName("second")]
        public int Second { get; set; }
        [JsonPropertyName("postId")]
        public int PostId { get; set; }
    }
}